relinker/src/main/java/com/getkeepsafe/relinker/ReLinkerInstance.java

```diff
@@ -74,7 +74,7 @@ public ReLinkerInstance force() {
     }
 
     /**
-     * Enables recursive library loading to resolve and load shared object -> shared object
+     * Enables recursive library loading to resolve and load shared object / shared object

```

Comment:
I'm confused about this change; it appears to alter the semantics of the comment in a way that isn't obviously correct to me.  What's the thought here?

Reply:
Before it's invalid javadoc. I guess we could use  &gt;

Reply:
Can't type that without a space or it substitutes the greater than 🤣

Reply:
lol
yeah, let's use the escape sequence here please.

Reply:
done